ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -56,6 +56,7 @@ releases:
       version: "4.12"
   latest:
     prerelease:
+      architecture: amd64
```

Comment:
Thanks Jianlin for the fix! I noticed that we didn't set arch for some amd64 images in https://github.com/openshift/release/pull/53318/files and thought the default arch is amd64. What's the default value if unset? cc @JianLi-RH for awareness

Reply:
I think it defaults to amd64. And define it explicitly makes it more clear.

Reply:
Yeah, just to make it more clear to read.